jwst/regtest/test_NIRcam_level2a.py

```diff
@@ -1,100 +0,0 @@
-""" Test for the detector1 pipeline using NIRCam data. This should take an uncal
```

Comment:
So where did this test file go? I see that the NIRSpec_IRS2 test module was renamed and included below, but I don't see any replacement for this NIRCam test module.

Reply:
I deleted it.  We already have a detector1 pipeline test for NIRCam imaging data with better simulated input data in the chained pipelines test, `test_nircam_image.py`.